### PR TITLE
Update thecustomizewindows

### PR DESCRIPTION
**Pull request by AbhishekGhosh**
Let's see.

### CODE REVIEW

File: src/chrome/content/rules/thecustomizewindows.com.xml [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
+	<target host="thecustomizewindows.net" />
+	<target host="www.thecustomizewindows.net" />
+	
+	<securecookie host="^thecustomizewindows-3470\.e\.kxcdn\.com$" name=".+"/>
+	<securecookie host="^cdn\.goroost\.com$" name=".+"/>
+	<securecookie host="^www\.googletagservices\.com$" name=".+"/>
+	<securecookie host="^tpc\.googlesyndication\.com$" name=".+"/>
+	<securecookie host="^googleads\.g\.doubleclick\.net$" name=".+"/>
+	
+	<rule from="^http://(?:www\.)?thecustomizewindows\.net/" to="https://thecustomizewindows.com/"/>
```
Comment (J0WI) [inline src/chrome/content/rules/thecustomizewindows.com.xml]: Use `(www\.)?` instead on `(?:www\.)?`
Please also document why this redirect is needed.

File: src/chrome/content/rules/thecustomizewindows.com.xml [hunk 2]
```diff
@@ -1,38 +1,20 @@
...
+	<target host="www.thecustomizewindows.com" />
+	
+	<target host="thecustomizewindows.net" />
+	<target host="www.thecustomizewindows.net" />
+	
+	<securecookie host="^thecustomizewindows-3470\.e\.kxcdn\.com$" name=".+"/>
+	<securecookie host="^cdn\.goroost\.com$" name=".+"/>
+	<securecookie host="^www\.googletagservices\.com$" name=".+"/>
+	<securecookie host="^tpc\.googlesyndication\.com$" name=".+"/>
+	<securecookie host="^googleads\.g\.doubleclick\.net$" name=".+"/>
```
Comment (J0WI) [inline src/chrome/content/rules/thecustomizewindows.com.xml]: Please do not set securecookie rules that don't depend on any target host in this ruleset.

Comment (AbhishekGhosh) [inline src/chrome/content/rules/thecustomizewindows.com.xml]: Thanks for the comment, `securecookie` rules will be removed on next update.

**Comment by jeremyn** [comment]
@AbhishekGhosh Are you still interested in working on this?

**Comment by AbhishekGhosh** [comment]
@jeremyn Thanks for the comment. Yes, it is important for me as the site is itself for new developers (so that I can use as example). But as it is listed on HSTS Preload list so it got pruned later. If someone adds it with minimal ruleset for main domain thecustomizewindows.com, I will be glad. As it was removed as community decision, it will look harsh if I again send pull request.

**Comment by jeremyn** [comment]
It looks like `thecustomizewindows.com` is HSTS preloaded, as you say, so we don't want to add a ruleset for it. `thecustomizewindows.net` is a technically valid target but it just redirects to `thecustomizewindows.com` and it's not worth having a ruleset just for that. As @J0WI wrote, the `securecookie`s aren't appropriate since they are for different domains. So there's nothing to do here.

I'm not sure what you mean in your comment. What minimal ruleset do you mean? Also I'm not sure what you mean by something looking "harsh".

**Comment by AbhishekGhosh** [comment]
@jeremyn Yes, you are correct. Nothing to do in this case.

**Comment by jeremyn** [comment]
All right, thanks @AbhishekGhosh . I'm closing this PR.

**Pull request closed without merging**